# erro funciona corretamente

    Code
      funcao_pai()
    Condition <erro_endbr_teste>
      Error in `funcao_pai()`:
      ! teste
      * informacao

